src/main/java/seedu/guest/model/guest/Bill.java

```diff
@@ -0,0 +1,58 @@
+package seedu.guest.model.guest;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.guest.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents any additional amount of money owed by the Guest in the guest book.
+ * Guarantees: immutable; is valid as declared in {@link #isValidBill(String)}
+ */
+public class Bill {
+    public static final String MESSAGE_CONSTRAINTS =
+            "Bill should only contain numerical digits and a decimal point, and it should be a non-negative value.";
```

Comment:
Is it important to add that a 1 decimal point values are not accepted in the MESSAGE_CONSTRAINTS?

Reply:
Agreed, added!